react-frontend/src/components/Navigation/Nav.css

```diff
@@ -1,24 +1,48 @@
 .nav {
+  background-color: #f0f4f8;
+  padding: 15px 20px;
   display: flex;
-  justify-content: space-between;
+  justify-content: center;
   align-items: center;
+  box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
+  border-top: 1px solid #e0e0e0;
+  border-bottom: 1px solid #e0e0e0;
+}
+
+.nav-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  max-width: 1200px;
   width: 100%;
-  height: 10vh; /* Adjust as needed to ensure it takes 10% of the screen height */
-  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
 }
 
 .nav-link {
-  flex: 1;
-  text-align: center;
-  padding: 15px 0;
-  text-decoration: none;
   color: #333;
-  margin: 0 2.5%;
-  transition: background-color 0.3s;
+  text-decoration: none;
+  padding: 12px 24px;
+  margin: 0 10px;
+  border-radius: 4px;
+  transition: all 0.3s ease;
+  font-weight: 600;
+  font-size: 16px;
 }
 
-
+.nav-link:hover,
 .nav-link.active {
-  font-weight: bold;
-  border-color: #333;
+  color: #007bff;
+  background-color: rgba(0, 123, 255, 0.1);
+  transform: translateY(-2px);
 }
+
+@media (max-width: 768px) {
+  .nav, .nav-container {
```

Comment:
Please, use mobile-first approach

Reply:
Refactored in [b655d78](https://github.com/YaroslavKSE/AuctionWeb/pull/3/commits/b655d78aa4cf807c011c5b5bb5f81bf9be9e0e84)